generic/_generic.normalize.scss

```diff
@@ -245,19 +225,15 @@ svg:not(:root) {
    ========================================================================== */
 
 /**
- * 1. Change the font styles in all browsers (opinionated).
- * 2. Remove the margin in Firefox and Safari.
+ * Remove the margin in Firefox and Safari.
  */
 
 button,
 input,
 optgroup,
 select,
 textarea {
-  font-family: sans-serif; /* 1 */
-  font-size: 100%; /* 1 */
```

Comment:
Inuitcss does not cater for this line being removed, and so it ends up being set back to:

```css
font: 11px system-ui;
```

by the browser. 

Do we want to push this back on normalize? Or do we want to handle this internally?